rskj-core/build.gradle

```diff
@@ -217,6 +219,7 @@ task jar(type:Jar, overwrite: true, dependsOn: generateResources) {
     }
     from sourceSets.main.output.classesDirs
     from sourceSets.main.output.resourcesDir
+    from sourceSets.test.output.classesDirs //Just for TestSystemProperties
```

Comment:
could you explain a bit more why we need this? Does it mean that the final jar will contain test classes?

Reply:
Nice catch! Forgot to remove it, i added it when playing with benchmarking tool.